src/gretel_trainer/models.py

```diff
@@ -162,3 +164,36 @@ def __init__(
             max_header_clusters=max_header_clusters,
             enable_privacy_filters=enable_privacy_filters,
         )
+
+
+class GretelAmplify(_BaseConfig):
+    """
+    This model is able to generate large quantities of data from real-world data or synthetic data.
+
+    Note: this model doesn't currently support privacy filtering.
+
+    Args:
+        config (str/dict, optional): Either a string representing the path to the config on the local filesystem, a string representing a path to the default Gretel configurations, or a dictionary containing the configurations. Default: "synthetics/amplify", a default Gretel configuration for Amplify.
+        max_rows (int, optional): The number of rows of synthetic data to generate. Defaults to 50000
+        max_header_clusters (int, optional): Default: 50
+    """
```

Comment:
@MasonEgger Added these docs.

And here's the blog introducing Amplify: https://gretel.ai/blog/introducing-gretel-amplify

Reply:
Looks good! Feel free to merge!